decentraland/explorer-desktop
# fix: Introduce a single point for resolution changing

Make sure the index in the resolutions list is consistent (it was not) Clean-up the code, reintroduce refresh rate (as it matters)

## What does this PR change?
Fixes [#2885](https://github.com/decentraland/unity-renderer/issues/2885)

There were multiple entry points for resolution changing. At start-up resolution was not applied properly because different refresh rate was not taken into consideration and thus it was breaking indexation. 

- Since refresh rate is meaningful (and we can't really use max refresh rate as some displays may not support max resolution at max refresh rate), I added it to resolution selection (any objections?)
- I unified the entry point so now there is only one way for changing resolution
- I cleaned up the code as it contained a lot of redundancies that apparently after debugging showed no use.

...

## How to test the changes?
Change resolution and window mode back and forth, changes should be preserved between client sessions.

1. Go to: https://renderer-artifacts.decentraland.org/desktop/index.html?dcl://DESKTOP-BRANCH={branch_name}
2. ...

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
